src/app/content/components/Toolbar.tsx

```diff
@@ -121,44 +169,145 @@ const SearchPrintWrapper = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
+  overflow: visible;
+
+  ${theme.breakpoints.mobile(css`
+    height: 100%;
+    ${SearchInputWrapper} {
+      border: none;
```

Comment:
why put these styles here instead of actually on the `SearchInputWrapper`?

Reply:
because there are two search input wrappers, and these are the styles for the mobile one